runtimes/cpu/cpu.zig

```diff
@@ -13,10 +19,25 @@ pub fn load() !*const pjrt.Api {
         return error.Unavailable;
     }
 
+    var arena = std.heap.ArenaAllocator.init(std.heap.c_allocator);
+    defer arena.deinit();
+
+    var r_ = try runfiles.Runfiles.create(.{ .allocator = arena.allocator() }) orelse {
+        std.debug.panic("Unable to create Runfiles isntance.", .{});
```

Comment:
Could we have another way to pass `zml/runtimes/`, maybe through a flag ?

Reply:
Very doable yes, I'll check with @steeve 
Actually could be handy for all other runtimes as well when testing debug versions.